tests/pkg_tomsfastmath/README.md

```diff
@@ -0,0 +1,6 @@
+# `pkg_tomsfastmath` test
```

Comment:
While I appreciate the effort of trying to properly document the package (which you did) I think this README file should be removed for two reasons:

1. It does not provide any additional information over the `main.c` itself, which is already small and concise
2. Prose description of code has the danger of diverging from the actual implementation and thus containing misinformation